tests/test_telcameras_v2_ingress.py

```diff
@@ -12,11 +12,12 @@
 from peoplemeasurement.models import Sensors
 from telcameras_v2.ingress_parser import TelcameraParser
 from telcameras_v2.models import CountAggregate, Observation
+from telcameras_v2.serializers import ObservationSerializer
 
 log = logging.getLogger(__name__)
 timezone = pytz.timezone("UTC")
 
-AUTHORIZATION_HEADER = {'HTTP_AUTHORIZATION': f"Token {settings.AUTHORIZATION_TOKEN}"}
```

Comment:
Deze black formatting is prima, maar ik zou dit soort auto formatting changes voortaan wel in een aparte commit houden. Dan is het duidelijker in welke commit inhoudelijke changes zitten, en in welke alleen maar formatting changes.

Reply:
ben ik mee eens!